docs/UserGuide.md

```diff
@@ -259,182 +277,191 @@ Format: `find PREFIX KEYWORD [KEYWORD]…​`
 
 Assigns a contact to a `Wedding`.
 
-Format: `assign-wedding INDEX w/WEDDINGNAME`
+Format: `assign-wedding PERSON_NUMBER w/WEDDING_NAME` or `asw PERSON_NUMBER w/WEDDING_NAME`
 
-* Assigns a specified contact to the `Wedding` based on their `INDEX`.
+* Assigns a specified contact to the `Wedding` based on their `PERSON_NUMBER`.
+* `WEDDING_NAME` is case-insensitive.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​.
 * The `Wedding` must exists in WedLinker before it can be assigned.
 * If the `Wedding` does not exist, you can use `f/` to force the creation and assignment of the `Wedding`.
 
-### Edit Wedding details : `edit-wedding`
+### Editing Wedding details : `edit-wedding` or `ew`
 
 Edits the details of a `Wedding`.
 
-Format: `edit-wedding INDEX [w/WEDDINGNAME] [a/ADDRESS]`
+Format: `edit-wedding INDEX [w/WEDDING_NAME] [a/ADDRESS]` or `ew INDEX [w/WEDDING_NAME] [a/ADDRESS]`
 
-* Edits the specific `Wedding` at the INDEX when in [list-wedding](#listing-all-weddings-list-weddings) view.
+* Edits the specific `Wedding` at the INDEX when in [list-weddings](#listing-all-weddings-list-weddings) view.
 * The index **must be a positive integer** 1, 2, 3, …​.
 * Existing values in the specified fields will be overwritten with the specified values.
 
-### Unassign contacts from a Wedding : `unassign-wedding`
+### Unassigning contacts from a Wedding : `unassign-wedding` or `uw`
 
 Unassigns a contact from a `Wedding` in WedLinker.
 
-Format: `unassign-wedding INDEX w/WEDDINGNAME`
+Format: `unassign-wedding PERSON_NUMBER w/WEDDING_NAME` or `uw PERSON_NUMBER w/WEDDING_NAME`
 
-* Unassign a contact that is assigned in a `Wedding`.
+* Unassigns a contact that is assigned to a `Wedding`.
+* `WEDDING_NAME` is case-insensitive.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​.
 
-### Deleting a Wedding : `delete-wedding`
+### Deleting a Wedding : `delete-wedding` or `dw`
 
 Deletes a `Wedding` from WedLinker.
 
-Format: `delete-wedding w/WEDDINGNAME [f/]`
+Format: `delete-wedding w/WEDDING_NAME [f/]` or `dw w/WEDDING_NAME [f/]`
 
-* Deletes a `Wedding ` from WedLinker.
+* Deletes a `Wedding` from WedLinker.
+* `WEDDING_NAME` is case-insensitive.
 * The no contacts should be assigned to the `Wedding` before it is deleted.
-* If there are still contacts assigned, you can use `f/` to force the deletion of the `Wedding` and unassign all contacts.
+* If there are still contacts assigned, you can use `f/` to force the deletion of the `Wedding`. This will unassign all contacts from that `Wedding`.
 
 ## Task Features
 
-### Creating a Task : `create-task`
+### Creating a Task : `create-task` or `ctask`
 
 Creates a `Task` in WedLinker
 
-Format: `create-task tk/TASKDESCRIPTION [REMARKS]`
+Format: `create-task tk/TASK_DESCRIPTION [REMARKS]` or `ctask tk/TASK_DESCRIPTION [REMARKS]`
```

Comment:
I think maybe this can be changed to have the command format specified in PR #380 as this specifies the date fields more accurately. The format currently specified there is `create-task tk/TASK_DESCRIPTION [d/DATE] [d/DATE]` or `ctask tk/TASK_DESCRIPTION [d/DATE] [d/DATE]`

Reply:
yep changed in my latest commit